packages/samples/test/output/testserver/body-boolean/openapi.json

```diff
@@ -157,9 +157,6 @@
               }
             }
           },
-          "204": {
```

Comment:
Seems like this response type: https://github.com/microsoft/cadl/blob/main/packages/samples/testserver/body-boolean/body-boolean.cadl#L13

Should result in a 200 / boolean and a 204 response, or some other alternative that allows a json null (which, I believe, boolean does not)

Reply:
Yes I agree, I filed an issue for this linked in the description as this isn't really the scope of this PR.